src/quo2/components/settings/accounts/view.cljs

```diff
@@ -14,7 +14,7 @@
 (defn avatar
   [avatar-props]
   [rn/view {:style (style/avatar-border)}
-   [account-avatar/account-avatar (assoc avatar-props :size 48)]])
+   [account-avatar/view (assoc avatar-props :size 48)]])
```

Comment:
The `Account Avatar` component is used in this component `settings > accounts`. I checked for usage across the codebase, but nothing was found.

It was introduced in this PR: https://github.com/status-im/status-mobile/pull/14643. I checked for the designs attached to this PR, looks like it was removed.